expyfun/_sound_controllers/_sound_controller.py

```diff
@@ -103,6 +109,10 @@ def __init__(self, params, stim_fs, n_channels=2, trigger_duration=0.01,
         self._id_after_onset = (
             str(params['SOUND_CARD_TRIGGER_ID_AFTER_ONSET']).lower() == 'true')
         self._extra_onset_triggers = list()
+        if np.isscalar(params['SOUND_CARD_DRIFT_TRIGGER']):
+            self._drift_trigger_time = [params['SOUND_CARD_DRIFT_TRIGGER']]
+        else:
+            self._drift_trigger_time = list(params['SOUND_CARD_DRIFT_TRIGGER'])
```

Comment:
I like this, I'll try to put it in. I just need to add `drift_trigger = params['SOUND_CARD_DRIFT_TRIGGER']` before the first line here and modify the code further down slightly, right? In the assertion, should we also allow numpy arrays?

Reply:
> I just need to add drift_trigger = params['SOUND_CARD_DRIFT_TRIGGER'] before the first line here and modify the code further down slightly, right?

Yeah I think so. I didn't test the above code so it might need some other adjustments, too

>  In the assertion, should we also allow numpy arrays?

No I wouldn't, it's easy enough to have people do `list(my_array)` and it simplifies code at our end